script/deploy.s.sol

```diff
@@ -47,14 +47,26 @@ contract DeployScript is Script {
     }
 
     function deployBorrower(TinlakeRoot root) internal returns (address) {
+        address feedFab = address(0);
+        if (keccak256(abi.encodePacked(vm.envString("NAV_IMPLEMENTATION"))) == keccak256("PV")) {
+            feedFab = getOrDeployFab("navfeedPV.sol:NAVFeedPVFab");
```

Comment:
This was required for me to run the deploy script locally for me.

Reply:
In general I think it would make sense to use individual salts or one global salt for the Tinlake deployment.

What is the meaning  of `bytes32 salt = 0x00000000000000000000000000000000000000008b99e5a778edb02572010000;`

Maybe you do something like `bytes32 salt = keccak(abi.encoode("Tinlake")`